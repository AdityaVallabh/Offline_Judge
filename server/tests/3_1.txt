helloWorld
hElLo
wORLD
h4110_w0r1d
!@#$%^&*()
1234567890
`~-=[]\;',./
_+{}|:"<>?
