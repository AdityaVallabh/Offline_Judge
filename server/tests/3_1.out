HELLOWORLD
HELLO
WORLD
H4110_W0R1D
!@#$%^&*()
1234567890
`~-=[]\;',./
_+{}|:"<>?
